bin/migrate-check

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+python manage.py migrate --check
+python manage.py migrate_clickhouse --check
+python manage.py run_async_migrations --check
```

Comment:
I haven't looked at how the async migrations are run, but is there a race here in that we need the app to be up to be able to run async migrations?

Reply:
we're not running them, just checking that they are all good just run the code in the file.

Reply:
I understand that we're not running them, and just checking. I'm just thinking that is there a scenario where:

 1. add new migration
 2. try to deploy so you can run the migration
 3. can't deploy because the check is stopping us

Reply:
(maybe I misunderstand what --check does but I was assuming it would exit non-zero if there are migrations to be applied)

Reply:
Only if they must be completed before this PostHog version.